app/src/components/form/LModal.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import LButton from "../button/LButton.vue";
+
+type Props = {
+    isVisible: boolean;
+    heading: string;
+    bLeftText?: string;
+    bRightText?: string;
+};
+defineProps<Props>();
+
+const emit = defineEmits(["close", "b-left-click", "b-right-click"]);
+</script>
+
+<template>
+    <div :open="isVisible" @close="emit('close')">
```

Comment:
This is nice - giving the possibility to close the modal by clicking outside of the popup area.